### PR TITLE
better handle large files

### PR DESCRIPTION
**Pull request by williballenthin**
- suggest providing flag -a for non-PE files
  - suggest providing flag -a for large files
  - extract static strings from large files
  - warn users of truncated static strings for 4GB files on 32bit systems

### CODE REVIEW

File: floss/main.py [hunk 1]
```diff
@@ -390,42 +391,77 @@ def create_script(sample_file_path, ida_python_file, decoded_strings):
             raise e
     # TODO return, catch exception in main()
 
+KILOBYTE = 1024
+MEGABYTE = 1024 * KILOBYTE
+MAX_FILE_SIZE = 16 * MEGABYTE
```
Comment (williballenthin) [inline floss/main.py]: these should go at the top of the file

File: floss/main.py [hunk 2]
```diff
@@ -451,6 +487,9 @@ def print_stack_strings(extracted_strings, min_length, quiet=False):
         print("")
 
 
+SUPPORTED_FILE_MAGIC = set(["MZ"])
```
Comment (williballenthin) [inline floss/main.py]: should go at the top of the file

File: floss/main.py (hunk 1 above)
Comment (williballenthin) [inline floss/main.py]: i picked `MAX_FILE_SIZE` out of thin air. we can tweak as appropriate. but i think generating a viv workspace for a 16MB file will take a little bit.

**Comment by mr-tz** [comment]
From the most recent Travis tests:
```30 passed, 21 xfailed in 482.10 seconds```
vs.
```32 passed, 22 xfailed in 497.10 seconds```

**Pull request merged**